src/phoenix/server/api/mutations/span_annotations_mutations.py

```diff
@@ -0,0 +1,109 @@
+from phoenix.server.api.types.SpanAnnotation import SpanAnnotation, to_gql_span_annotation
+
+
+@strawberry.type
+class SpanAnnotationMutationPayload:
+    span_annotations: List[SpanAnnotation]
+
+
+@strawberry.type
+class SpanAnnotationMutationMixin:
+    @strawberry.mutation(permission_classes=[IsAuthenticated])  # type: ignore
+    async def create_span_annotations(
+        self, info: Info[Context, None], input: List[CreateSpanAnnotationsInput]
+    ) -> SpanAnnotationMutationPayload:
+        span_rowid = from_global_id_with_expected_type(input[0].span_id, "Span")
+        inserted_annotations: Sequence[models.SpanAnnotation] = []
+        async with info.context.db() as session:
+            values_list = [
+                dict(
+                    trace_rowid=span_rowid,
+                    name=annotation.name,
+                    label=annotation.label,
+                    score=annotation.score,
+                    explanation=annotation.explanation,
+                    annotator_kind=annotation.annotator_kind,
+                    metadata_=annotation.metadata,
+                )
+                for annotation in input
+            ]
+            stmt = (
+                insert(models.SpanAnnotation).values(values_list).returning(models.SpanAnnotation)
+            )
+            result = await session.scalars(stmt)
+            inserted_annotations = result.all()
+
+        return SpanAnnotationMutationPayload(
+            span_annotations=[
+                to_gql_span_annotation(annotation) for annotation in inserted_annotations
+            ]
+        )
+
+    @strawberry.mutation(permission_classes=[IsAuthenticated])  # type: ignore
+    async def patch_span_annotations(
+        self, info: Info[Context, None], input: List[PatchAnnotationsInput]
+    ) -> SpanAnnotationMutationPayload:
+        patched_annotations = []
+        async with info.context.db() as session:
+            for annotation in input:
+                span_annotation_id = from_global_id_with_expected_type(
+                    annotation.annotation_id, "SpanAnnotation"
+                )
+                patch = {
+                    column.key: patch_value
+                    for column, patch_value, column_is_nullable in (
+                        (models.SpanAnnotation.name, annotation.name, False),
+                        (models.SpanAnnotation.annotator_kind, annotation.annotator_kind, False),
+                        (models.SpanAnnotation.label, annotation.label, True),
+                        (models.SpanAnnotation.score, annotation.score, True),
+                        (models.SpanAnnotation.explanation, annotation.explanation, True),
+                        (models.SpanAnnotation.metadata_, annotation.metadata, False),
+                    )
+                    if patch_value is not UNSET and (patch_value is not None or column_is_nullable)
+                }
```

Comment:
```suggestion
                patch = {
                    column.key: patch_value
                    for column, patch_value in (
                        (models.SpanAnnotation.name, annotation.name),
                        (models.SpanAnnotation.annotator_kind, annotation.annotator_kind),
                        (models.SpanAnnotation.label, annotation.label),
                        (models.SpanAnnotation.score, annotation.score),
                        (models.SpanAnnotation.explanation, annotation.explanation),
                        (models.SpanAnnotation.metadata_, annotation.metadata),
                    )
                    if patch_value is not UNSET
                }
```

Does this accomplish the same thing?

Reply:
I'm pretty sure it might but I'm reusing an existing pattern

Reply:
I'm not positive about the behavior of UNSET so I'm going to leave it for now